### PR TITLE
Add unicode-fonts as submodule

### PR DESCRIPTION
**Pull request by ylemkimon**
\+ Removed checkout process from screenshotter.sh.

I think it's better to have them as a submodule, since git and GitHub handle it very well and we already have one submodule. Also, it's cross-platform and no extra manual step is needed when running `screenshotter.js` directly.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,6 @@
 [submodule "submodules/katex-fonts"]
 	path = submodules/katex-fonts
 	url = https://github.com/KaTeX/katex-fonts
+[submodule "submodules/unicode-fonts"]
+	path = submodules/unicode-fonts
+	url = https://github.com/Khan/KaTeX-test-fonts.git
```
Comment (kevinbarabash) [inline .gitmodules]: I can move this repo to the `KaTeX` organization.

Comment (kevinbarabash) [reply .gitmodules]: It probably makes it sense to have it at `KaTeX/test-fonts` or `KaTeX/unicode-test-fonts`, wdyt?

Comment (edemaine) [reply .gitmodules]: I think either is fine.

Comment (ylemkimon) [reply .gitmodules]: As its current name is `KaTeX-test-font` and in the future maybe we can add other ASCII fonts support, I move for the former.

**Comment by kevinbarabash** [comment]
@ylemkimon I've move the test font repo to https://github.com/KaTeX/katex-test-fonts.  Can you change the submodule folder to `katex-test-fonts` when you update the url?

**Pull request merged**